client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -181,6 +232,7 @@ export default connect( state => {
 	const selectedSiteId = getSelectedSiteId( state );
 
 	return {
+		customizeUrl: getCustomizerUrl( state, selectedSiteId ),
```

Comment:
Minor: Having them called the same might be a bit more consistent (`customizerUrl`)

Reply:
This is [copied](https://github.com/Automattic/wp-calypso/blob/cf063a41fa3ab80ba3debc6fe78cf2789405c9b4/client/my-sites/sidebar/sidebar.jsx#L689) and shows up like this in a few places. I think it makes okish sense since the URL is for the "customize" link.